### PR TITLE
Test sorting products by price

### PR DESCRIPTION
**Pull request by elwoodxblues**
A follow-up to PR #1874 which adds a test for sorting products by price in API.

### Pull Request Checklist

1. [x] Privileged views and APIs are guarded by proper permission checks.
1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Python code quality checks pass: `pycodestyle`, `pydocstyle`, `pylint`.
1. [x] JavaScript code quality checks pass: `eslint`.

### CODE REVIEW

File: tests/test_graphql.py [hunk 1]
```diff
@@ -234,6 +234,49 @@ def test_filter_product_by_attributes(client, product_in_stock):
     assert product_data['name'] == product_in_stock.name
 
 
+def test_sort_products(client, product_in_stock):
+    # set price of the first product
+    product_in_stock.price = Money('10.00', 'USD')
```
Comment (rafalp) [inline tests/test_graphql.py]: The convention used in our tests (and by extension whole codebase) is explicit`currency=` for money's currency.

Comment (patrys) [reply tests/test_graphql.py]: Isn't this because `currency` was in fact the third parameter to `Price`?

Comment (rafalp) [reply tests/test_graphql.py]: Yep, may be a remnant of those times that sticked around. May be good to do a follow-up PR that drops the superficial `currency=` from our codebase then?

Comment (patrys) [reply tests/test_graphql.py]: Yup! I think explicitly naming parameters is useful when either the order of arguments is not clear or the argument itself is a magic value (like `True` or `7`).

**Pull request merged**